index.php

```diff
@@ -0,0 +1,14 @@
+<?php
+require_once 'DBconnect/DBconnect.php';
+require_once 'DBgetAllData/DBgetAllData.php';
+
+$db = DBconnect();
+$allTableData =DBgetAllData($db);
+
+require_once 'DisplayLocations/DisplayLocations.php';
+
+Echo DisplayLocations($allTableData);
+
+
+
+
```

Comment:
Should have some sort of valid HTML to output.

Ash's previous comment about this is not resolved, don't mark it resolved when it isn't. The 'resolve' button is for the reviewer to accept your change as a resolution to their comment, not you :)

Reply:
added